pwa/app/(common)/docs/[...slug]/page.tsx

```diff
@@ -7,18 +7,27 @@ import {
 import classNames from "classnames";
 import { current, versions } from "consts";
 import Script from "next/script";
-import { Metadata, ResolvingMetadata } from "next";
+import { Metadata } from "next";
 import BreadCrumbs from "components/docs/BreadCrumbs";
+import { notFound } from "next/navigation";
+import { getVersionAndSlugFromSlugs } from "utils";
 
 export async function generateStaticParams() {
   const slugs: { slug: string[] }[] = [];
   const navs = await loadV2DocumentationNav(current);
   for (const nav of navs) {
     for (const link of nav.links) {
-      slugs.push({ slug: link.link.replace("/docs/", "").split("/") });
+      slugs.push({
+        slug: link.link
+          .replace("/docs/", "")
+          .split("/")
+          .filter((p) => p !== ""),
```

Comment:
```suggestion
          .filter((p) => !p),
```
mais bon en vrai ce que t'as mis c'est plus explicite